projects/frr/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://frrouting.org"
+language: c++
+primary_contact: "security@lists.frrouting.org"
+auto_ccs:
+- "p.antoine@catenacyber.fr"
+fuzzing_engines:
+ - libfuzzer
+ - afl
```

Comment:
Earlier on you've built FRR for libfuzzer, which if I recall correctly, excludes the AFL entrypoints (libfuzzer provides its own `main` and the AFL entrypoints are in our `main`, so that's compiled out when libfuzzer is enabled).

Reply:
The way oss-fuzz works is to have the same libFuzzer entry points and run them with afl with a specific afl driver.
Do I get it correct @asraa ?